README.md

```diff
@@ -20,6 +28,27 @@ pnpm dev
 
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
 
+### Troubleshooting
+If you encounter any issues during setup, refer to the following troubleshooting tips:
+
+- For new contributors you may encounter the error
+  ```
+   GITHUB_PAT is not configured in the environment.
+   Request quota exhausted for request POST /graphql
+  ```
+  In place of running `pnpm dev` create you own GitHub access token. [Read Steps here](https://docs.github.com/en/enterprise-server@3.9/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
+  Then run the following command instead:
+  ```
+  GITHUB_PAT=<YOUR_KEY> pnpm dev
+  ```
```

Comment:
You can also mention about using auth token provided by github's cli if user has gh cli installed that way contributor need not create a PAT token...

<img width="814" alt="image" src="https://github.com/coronasafe/leaderboard/assets/25143503/95870fc2-4be2-407a-b1f0-fceda8e91c72">

Reply:
Ok got it. This is better than directly putting the key.